outputs.tf

```
@@ -8,7 +8,12 @@ output "route53resourcechange_role" {
   description = "IAM role that allows sufficient permissions to modify resource records in the cyber.dhs.gov zone."
 }
 
+output "sesmanagementsuppressionlist_role" {
```

Not sure if this was a typo or intentional, but it seems (to me, at least) that this is more in line with the other output name:
```suggestion
output "sesmanagesuppressionlist_role" {
```

If you choose to accept this change, don't forget to update the output name in the README as well.

---

Strong eye!  See commit ff55a4c.